lib/krane/kubernetes_resource/stateful_set.rb

```diff
@@ -5,6 +5,9 @@ class StatefulSet < PodSetBase
     TIMEOUT = 10.minutes
     ONDELETE = 'OnDelete'
     SYNC_DEPENDENCIES = %w(Pod)
+    REQUIRED_ROLLOUT_ANNOTATION = "required-rollout"
+    REQUIRED_ROLLOUT_TYPES = %w(full none).freeze
```

Comment:
yep! more specifically, the `freeze` method is what makes the array immutable -- the `%w` notation just creates the array of strings. we're using this syntax because most string arrays in krane seem to follow this pattern as well